clusters/sandbox-values.yaml

```diff
@@ -4,17 +4,51 @@ global:
     enabled: true
     ip: "10.101.29.134"
 
-httpsEgressSafelist:
-- name: sqs-eu-west-2
-  fqdn: sqs.eu-west-2.amazonaws.com
-- name: queue-eu-west-2
-  fqdn: eu-west-2.queue.amazonaws.com # legacy endpoint
-- name: stub-connector
-  fqdn: test-connector.london.sandbox.govsvc.uk
-- name: test-intgegration
-  fqdn: test-integration-connector.london.sandbox.govsvc.uk
-- name: hub-integration
-  fqdn: www.integration.signin.service.gov.uk
+egressSafelist:
+- hosts:
+    - "sqs.eu-west-2.amazonaws.com"
+    - "eu-west-2.queue.amazonaws.com"
+  ports:
+  - name: sqs-http
+    number: 80
+    protocol: HTTP
+  - name: sqs-https
+    number: 443
+    protocol: TLS
+  location: MESH_EXTERNAL
+  resolution: DNS
+
+- hosts: ["*.rds.amazonaws.com"]
+  ports:
+  - name: rds
+    number: 5432
+    protocol: TCP
+  location: MESH_EXTERNAL
+  resolution: DNS
+
+- hosts: ["test-connector.london.sandbox.govsvc.uk"]
+  ports:
+  - name: test-connector-sandbox-https
```

Comment:
these port names can just be `https` they don't have to have the entire name dup'd

Reply:
Done.